test/testing_extensions_test.rb

```diff
@@ -30,7 +30,7 @@ def self.doit(arg)
 
 describe Sidekiq::DelayExtensions do
   before do
-    Sidekiq.redis { |c| c.flushdb }
+    @cfg = reset!
```

Comment:
this file no longer has an equivalent in Sidekiq. Fortunately, it mostly just works.